docs/appengine/taskqueue/push/defer.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package counter
+
+import (
+	"context"
+
+	"google.golang.org/appengine/delay"
+)
+
+func example() {
+	var ctx context.Context
```

Comment:
This should be `ctx := context.Background` or something similar. Using `ctx` as is might mislead readers, since it's a nil interface and needs a backing type.

Reply:
Done.